v2/src/actors/cal_get_promise.js

```diff
@@ -60,40 +58,40 @@ function processResult(lines, dayOnly) {
   return final;
 }
 
-module.exports = function() {
-  this.getCalStr = function(debugMode, dayOnly) {
-    return new Promise((resolve, reject) => {
-      var request = new XMLHttpRequest();
-      url = debugMode
-        ? "https://www.calendarlabs.com/ical-calendar/ics/71/Sweden_Holidays.ics"
-        : "https://www.nordiskamuseet.se/calendar/ical/ical/calendar-nordiska-museet.ics";
-      request.open("GET", url, true);
-      var today = new Date();
-      var date_now_f =
-        today.getFullYear() +
-        "-" +
-        ("0" + (today.getMonth() + 1)).slice(-2) +
-        "-" +
-        ("0" + today.getDate()).slice(-2);
-      request.send(null);
-      HAR_HITTAT_VEVENT = false;
-      found_min_one = false;
-      request.onreadystatechange = function() {
-        //checks if response is ready.
-        if (request.readyState === 4 && request.status === 200) {
-          var type = request.getResponseHeader("Content-Type");
+export const getCalStr = function(debugMode, dayOnly) {
+  return new Promise((resolve, reject) => {
+    var request = new XMLHttpRequest();
+    url = debugMode
+      ? "https://www.calendarlabs.com/ical-calendar/ics/71/Sweden_Holidays.ics"
+      : "https://www.nordiskamuseet.se/calendar/ical/ical/calendar-nordiska-museet.ics";
+    request.open("GET", url, true);
+    var today = new Date();
+    var date_now_f =
+      today.getFullYear() +
+      "-" +
+      ("0" + (today.getMonth() + 1)).slice(-2) +
+      "-" +
+      ("0" + today.getDate()).slice(-2);
+    request.send(null);
+    HAR_HITTAT_VEVENT = false;
+    found_min_one = false;
+    request.onreadystatechange = function() {
+      //checks if response is ready.
+      if (request.readyState === 4 && request.status === 200) {
+        var type = request.getResponseHeader("Content-Type");
 
-          if (type.indexOf("text") !== 1) {
-            var lines = request.responseText.split("\n");
+        if (type.indexOf("text") !== 1) {
+          var lines = request.responseText.split("\n");
 
-            //this is where we process the data given by request.
-            final = processResult(lines, dayOnly);
-            resolve(final);
-            // END OF: if (type.indexOf("text") !== 1)
-          }
-          // END OF: if (request.readyState === 4 && request.status === 200)
+          console.log(lines);
```

Comment:
I added this line here to show that the data is downloading correctly, but is being processed incorrectly.

<img width="1112" alt="Screen Shot 2020-05-03 at 11 01 31 AM" src="https://user-images.githubusercontent.com/1957315/80917841-82766d00-8d2f-11ea-81c2-ef0bb64ac55e.png">